fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -96,6 +96,11 @@ def self.windows?
       (/cygwin|mswin|mingw|bccwin|wince|emx/ =~ RUBY_PLATFORM) != nil
     end
 
+    def self.command?(name)
+      `which #{name} 2> /dev/null`
```

Comment:
I think `command -v #{name.shell_escape} 2>/dev/null` is more portable / recommended to check if a command exists rather than `which`? (I don't remember where I read that, but I did 😄 )

Reply:
I wonder if that will work in WIndows. I can give it a try.

Reply:
done. We'll see what happens.